docs/content/dropbox.md

```diff
@@ -492,6 +492,25 @@ Properties:
 - Type:        string
 - Required:    false
 
+#### --dropbox-export-formats
+
+Comma separated list of preferred formats for exporting files
```

Comment:
This part of the file is auto generated from the `export_formats` definition, so put this nice help in there.

Reply:
Oh fascinating, I'll do that.